go.mod

```diff
@@ -536,3 +537,9 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/hashicorp/raft => /Users/swenson/projects/raft
+
+replace github.com/hashicorp/raft-boltdb => /Users/swenson/projects/raft-boltdb
+
+replace github.com/hashicorp/raft-boltdb/v2 => /Users/swenson/projects/raft-boltdb/v2
```

Comment:
@swenson I think these may have made it in by accident.

Reply:
It's more of a reminder that I still need to merge those before this is done.